### ISSUE TITLE
some tasks run very frequently and do not want to log. I want to configure logging separately for tasks with long running intervals

### ISSUE THREAD

**Issue by Fdemo123** [apache/shardingsphere-elasticjob#1958]
some tasks run very frequently and do not want to log. I want to configure logging separately for tasks with long running intervals

**Comment by sunkai-cai** [comment]
For English only.

Are you using `elasticjob-lite-spring-boot-starter` package?  

This a good idea, maybe we can support it in next version.

**Comment by TeslaCN** [comment]
Please translate your issue into English.

**Comment by Fdemo123** [comment]
> For English only.
> 
> Are you using `elasticjob-lite-spring-boot-starter` package?
> 
> This a good idea, maybe we can support it in next version.

I am using elastic job version 2.1.5. I have written a springboot starter to handle the loading of tasks and control whether each task records logs separately. I see that the official website is updated. I want to upgrade and find that this function is not available. I hope the official can support it. In this way, the developer does not need to do secondary encapsulation.

**Comment by sunkai-cai** [comment]
> > For English only.
> > Are you using `elasticjob-lite-spring-boot-starter` package?
> > This a good idea, maybe we can support it in next version.
> 
> I am using elastic job version 2.1.5. I have written a springboot starter to handle the loading of tasks and control whether each task records logs separately. I see that the official website is updated. I want to upgrade and find that this function is not available. I hope the official can support it. In this way, the developer does not need to do secondary encapsulation.

Of course,  the features in the plan, and welcome the developer submit PR.  Could you contribute code to complete the plan faster .

**Comment by TeslaCN** [comment]
Event trace is not a global configuration. You can configure event trace for each job separately.
